packages/client.api/src/object/FetchPageResult.ts

```diff
@@ -18,24 +18,23 @@ import type {
   ObjectOrInterfaceDefinition,
   ObjectOrInterfacePropertyKeysFrom2,
 } from "@osdk/api";
-import type { IsNever } from "type-fest";
 import type { DefaultToFalse } from "../definitions/LinkDefinitions.js";
-import type { Osdk } from "../OsdkObjectFrom.js";
+import type { IsNever, Osdk } from "../OsdkObjectFrom.js";
 import type { PageResult } from "../PageResult.js";
 import type { NullabilityAdherence } from "./FetchPageArgs.js";
 
-/** @internal exposed for a test */
+/** exposed for a test */
 export type RespectNullability<S extends NullabilityAdherence> = S extends false
```

Comment:
Wait why are we exposing these types?

Reply:
Because anything marked @internal is no longer being included in the output. This is removing things like references to `@osdk/internal.*` from `@osdk/client`.